agent-langgraph/README.md

```diff
@@ -0,0 +1,232 @@
+
+   ```bash
+   DATABRICKS_CONFIG_PROFILE="DEFAULT" # change to the profile name you chose
+   ```
+
+   **Option 2: Personal Access Token (PAT)**
+
+   See the [PAT documentation](https://docs.databricks.com/aws/en/dev-tools/auth/pat#databricks-personal-access-tokens-for-workspace-users).
+
+   ```bash
+   # Add these to your .env.local file
+   # DATABRICKS_HOST="https://host.databricks.com"
+   # DATABRICKS_TOKEN="dapi_token"
+   ```
+
+   See the [Databricks SDK authentication docs](https://docs.databricks.com/aws/en/dev-tools/sdk-python#authenticate-the-databricks-sdk-for-python-with-your-databricks-account-or-workspace).
+
+3. **Create and link an MLflow experiment to your app**
+
+   Create an MLflow experiment to enable tracing and version tracking. This is automatically done by the `./scripts/quickstart.sh` script.
+
+   Create the MLflow experiment via the CLI:
+
+   ```bash
+   DATABRICKS_USERNAME=$(databricks current-user me | jq -r .userName)
+   databricks experiments create-experiment /Users/$DATABRICKS_USERNAME/agents-on-apps
+   ```
+
+   Make a copy of `.env.example` to `.env.local` and update the `MLFLOW_EXPERIMENT_ID` in your `.env.local` file with the experiment ID you created. The `.env.local` file will be automatically loaded when starting the server.
+
+   ```bash
+   cp .env.example .env.local
+   # Edit .env.local and fill in your experiment ID
+   ```
+
+   See the [MLflow experiments documentation](https://docs.databricks.com/aws/en/mlflow/experiments#create-experiment-from-the-workspace).
+
+4. **Test your agent locally**
+
+   Start up the agent server and chat UI locally:
+
+   ```bash
+   ./scripts/start-app.sh
+   ```
+
+   Query your agent via the UI (http://localhost:8000) or REST API:
+
+   **Advanced server options:**
+
+   ```bash
+   uv run start-server --reload   # hot-reload the server on code changes
+   uv run start-server --port 8001 # change the port the server listens on
+   uv run start-server --workers 4 # run the server with multiple workers
+   ```
+
+   - Example streaming request:
+     ```bash
+     curl -X POST http://localhost:8000/invocations \
+     -H "Content-Type: application/json" \
+     -d '{ "input": [{ "role": "user", "content": "hi" }], "stream": true }'
+     ```
+   - Example non-streaming request:
+     ```bash
+     curl -X POST http://localhost:8000/invocations  \
+     -H "Content-Type: application/json" \
+     -d '{ "input": [{ "role": "user", "content": "hi" }] }'
+     ```
+
+## Modifying your agent
+
+See the [LangGraph documentation](https://docs.langchain.com/oss/python/langgraph/quickstart) for more information on how to edit your own agent.
+
+Required files for hosting with MLflow `AgentServer`:
+
+- `agent.py`: Contains your agent logic. Modify this file to create your custom agent.
+- `start_server.py`: Initializes and runs the MLflow `AgentServer` with agent_type="ResponsesAgent".
```

Comment:
```suggestion
- `start_server.py`: Initializes and runs the MLflow `AgentServer`. You don't have to modify this file for most common use cases, but can customize server behavior here. For example, you add additional server routes (e.g. a custom `/metrics` endpoint or `/ops` UI) in this file, or update the `agent_type` of the `AgentServer` if you'd like to implement a custom agent API interface (see docs).
```

Is it useful to mention `agent_type` here? Or should we just mention it in the code comments itself? It seems like a detail that people shouldn't usually need to care about? Is it documented somewhere in OSS docs (coudln't find it from a quick search)?